pkg/tar/tar.go

```diff
@@ -307,6 +263,14 @@ func (t *stiTar) writeTarHeader(tarWriter *tar.Writer, dir string, path string,
 	if err != nil {
 		return err
 	}
+	// on Windows, tar.FileInfoHeader incorrectly interprets directory symlinks
+	// as directories.  https://github.com/golang/go/issues/17541
```

Comment:
And I thought Windows had no symlinks... `.lnk` yes, but "directory symlinks", that's new :)